src/libraries/pretest.proj

```diff
@@ -26,9 +26,7 @@
     <ProjectReference Include="$(MonoProjectRoot)wasm\symbolicator\WasmSymbolicator.csproj" Condition="'$(TargetOS)' == 'browser'" />
 
     <!-- needed to test workloads for wasm -->
-    <ProjectReference Include="$(InstallerProjectRoot)pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj"
-                      Pack="true"
-                      Condition="'$(TargetOS)' == 'browser' or '$(TargetOS)' == 'wasi' or '$(TestReadyToRun)' == 'true'" />
+    <ProjectReference Include="$(InstallerProjectRoot)pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.sfxproj" Pack="true" Condition="'$(TargetOS)' == 'browser' or '$(TargetOS)' == 'wasi'" />
```

Comment:
Revert putting everything into a single line.

Reply:
I'm much in favor of doing it in multiple lines like I had it, but I thought you would prefer if I'd left it untouched since I'm no longer editing that piece of code.